### ISSUE TITLE
Icebox Plasma Rivers killing Plastitanium Golems

### ISSUE THREAD

**Issue by Tanden22** [tgstation/tgstation#62667]
## Reproduction:
-be Plastitanim Golem from Free Golem Ship
-walk in plasma river
-take hella damage everywhere
-die if you stand in it too long

I don't _think_ this is supposed to happen, nor do i know if mining borgs with the lavaproofing upgrade or Clarkes are immune, but i figure i'd throw this here anyways.

**Comment by timothymtorres** [comment]
https://github.com/tgstation/tgstation/blob/b79dd74fd0aef748a8e54c45402cebb3cd98e1c6/code/modules/mob/living/carbon/human/species_types/golems.dm#L260-L274

There is a similar issue at #51505 where it points out that a lava river is not the same thing as a plasma river.  A plasma river requires the FREEZE_PROOF.

**Comment by Tanden22** [comment]
Ah, I see.  Is there even anything _with_ FREEZE_PROOF yet?

**Comment by mixzombie** [comment]
This is why all Miners Hate Icebox. Because nothing it tooled for it. No Freeze protection gear outside of basic winter jacket. Which offer basically no protection.

**Comment by Tanden22** [comment]
I've literally never seen anyone complain about the cold. It's always complaints about the fauna.

**Comment by mixzombie** [comment]
Well Icebox was never a finished project it was an experiment as far as i remember. Plasma rivers are cool conceptually and all but there are no bypass's for it like with lava. Drake armor which is seemingly readily available on Ice box is a trap. A lure to suck in dumb first time miners and they are punished for it, in the form of cold (burn damage) just for running around with it. Heaven forbid a lizard player try it out as they will not make it back to the station.

; TLDR please just give drake armor the same cold protection as regular miner suit

**Comment by Tanden22** [comment]
Mate, drake armor isn't a trap. I've never seen anyone take cold damage while wearing it in the caves, and I've never seen lizard miners complain about the cold damage either. You looking at old info?

**Comment by redsunoverparadise** [comment]
Ash Drake armour has had cold resist for quite a while now

**Comment by Sealed101** [comment]
https://github.com/tgstation/tgstation/pull/74197 killed plastitanium golems, and plasma buff does not stop the plasma rivers. uh oh
